docs/metrics.md

```diff
@@ -27,6 +27,9 @@ Current balloon bytes.
 ### kubevirt_vmi_memory_available_bytes
 Amount of `usable` memory as seen by the domain.
 
+### kubevirt_vmi_memory_domain_total_bytes
+The amount of memory in bytes used by the domain.
```

Comment:
Maybe also add that it is a static valume representing the upper bound.

Reply:
It is not the maximum amount. It is  `memory` in virDomainInfo See https://libvirt.org/html/libvirt-libvirt-domain.html#virDomainInfo

What about this one?
`The amount of memory in bytes used by the domain. The 'memory' value in domain xml file.`

Reply:
I find the current description still confusing. I'd avoid `used by the domain` (similarly, we're avoiding this term in the metric's name). I think `allocated` is much clearer, and conveys the (usually) static-ness of this value. How about this phrasing:
```
The amount of memory in bytes allocated to the domain. The 'memory' value in domain xml file.`
```

Reply:
I agree with Zvi, although the official documentation says `the memory in KBytes used by the domain`, it is confusing since the `used` memory may be less than the `allocated` memory. But what does this metric show is how much memory libvirt has allocated to the domain.